components/admin_console/admin_definition.jsx

```diff
@@ -4117,7 +4118,11 @@ const AdminDefinition = {
                         type: Constants.SettingsTypes.TYPE_CUSTOM,
                         component: OpenIdConvert,
                         key: 'OpenIdConvert',
-                        isHidden: () => true,
+                        isHidden: it.any(
```

Comment:
Should this be like this? Namely, only hidden if *none* of them are licensed?
```suggestion
                        isHidden: it.all(
```

Reply:
Removed check on Google and Office365 completely. Now only check for "OpenId" license. 
So it is hidden if it is NOT Licensed for "OpenId".
It is also hidden if it is licensed for OpenId and does NOT use LegacyOAuth.